src/Knp/Menu/ItemInterface.php

```diff
@@ -8,6 +8,8 @@
  * It roughly represents a single <li> tag and is what you should interact with
  * most of the time by default.
  * Originally taken from ioMenuPlugin (http://github.com/weaverryan/ioMenuPlugin)
+ *
+ * @extends \IteratorAggregate<self>
```

Comment:
Shouldn't this be `IteratorAggregate<TKey, TValue>`?

```suggestion
 * @extends \IteratorAggregate<int, self>
```

https://github.com/phpstan/phpstan-src/blob/fe4c770254b49d64816e4b62c2b6d109f88e0832/stubs/iterable.stub#L11-L25

Reply:
actually, keys are strings as they are the names of children.

Reply:
Right, I've updated my suggestion. Thanks.

Reply:
Thanks @stof @derrabus ! I've commited the suggestion!